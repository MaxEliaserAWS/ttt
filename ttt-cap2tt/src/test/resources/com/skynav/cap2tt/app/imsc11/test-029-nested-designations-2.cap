Lambda字幕V4	DF1+1	SCENE"和文標準"

1	00000218/00000616	（アナウンス）
				＠斜３［二陣戦 東 ＠ルビ下［辻｜つじ］＠君　西 ＠ルビ下［澤井｜さわい］＠君］＠
2	00012115/00012403	（ナレーション）	＠縦右	＠行頭
				＠斜３［声優］＠
				＠斜３［小野＠ルビ左［大輔｜だいすけ］＠と下野 ＠ルビ左［紘｜ひろ］＠が］＠
3	00013118/00013422	（ナレーション）
				＠斜３［＠ルビ下［幽助｜ゆうすけ］＠は＠ルビ下［桑原｜くわばら］＠･＠ルビ下［飛影｜ひえい］＠･＠ルビ下［蔵馬｜くらま］＠たちと共に］＠
4	00094723/00095008	（桐仁）＠斜３［＠ルビ右［頭捻｜ずぶね］＠り　＠ルビ右［渦切｜うずきり］＠！］＠	＠縦左	＠行頭
5	00261310/00261426	（ナレーター）	＠縦左	＠行頭
				＠斜３［原作者 ＠ルビ左［久住｜くすみ］＠＠ルビ左［昌之｜まさゆき］＠が］＠
