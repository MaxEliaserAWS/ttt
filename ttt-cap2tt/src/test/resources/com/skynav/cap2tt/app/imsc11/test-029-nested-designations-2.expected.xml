<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="center"/><style xml:id="s2" tts:textAlign="start"/><style xml:id="s3" tts:fontShear="16.78842%"/><style xml:id="s4" tts:ruby="container"/><style xml:id="s5" tts:ruby="base"/><style xml:id="s6" tts:ruby="text" tts:rubyPosition="after"/><style xml:id="s7" tts:ruby="text" tts:rubyPosition="before"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/><region xml:id="縦左" tts:displayAlign="after" tts:extent="15rw 80rh" tts:position="left 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:00:02:18" end="00:00:06:16" style="s1"><span style="s2">（アナウンス）<br/><span style="s3">二陣戦 東 </span><span style="s3"><span style="s4"><span style="s5">辻</span><span style="s6">つじ</span></span></span><span style="s3">君　西 </span><span style="s3"><span style="s4"><span style="s5">澤井</span><span style="s6">さわい</span></span></span><span style="s3">君</span></span></p><p begin="00:01:21:15" end="00:01:24:03" region="縦右">（ナレーション）<br/><span style="s3">声優</span><br/><span style="s3">小野</span><span style="s3"><span style="s4"><span style="s5">大輔</span><span style="s6">だいすけ</span></span></span><span style="s3">と下野 </span><span style="s3"><span style="s4"><span style="s5">紘</span><span style="s6">ひろ</span></span></span><span style="s3">が</span></p><p begin="00:01:31:18" end="00:01:34:22" style="s1"><span style="s2">（ナレーション）<br/><span style="s3"><span style="s4"><span style="s5">幽助</span><span style="s6">ゆうすけ</span></span></span><span style="s3">は</span><span style="s3"><span style="s4"><span style="s5">桑原</span><span style="s6">くわばら</span></span></span><span style="s3">･</span><span style="s3"><span style="s4"><span style="s5">飛影</span><span style="s6">ひえい</span></span></span><span style="s3">･</span><span style="s3"><span style="s4"><span style="s5">蔵馬</span><span style="s6">くらま</span></span></span><span style="s3">たちと共に</span></span></p><p begin="00:09:47:23" end="00:09:50:08" region="縦左">（桐仁）<span style="s3"><span style="s4"><span style="s5">頭捻</span><span style="s7">ずぶね</span></span></span><span style="s3">り　</span><span style="s3"><span style="s4"><span style="s5">渦切</span><span style="s7">うずきり</span></span></span><span style="s3">！</span></p><p begin="00:26:13:10" end="00:26:14:26" region="縦左">（ナレーター）<br/><span style="s3">原作者 </span><span style="s3"><span style="s4"><span style="s5">久住</span><span style="s6">くすみ</span></span></span><span style="s3"><span style="s4"><span style="s5">昌之</span><span style="s6">まさゆき</span></span></span><span style="s3">が</span></p></div></body></tt>